x-pack/plugins/data_usage/common/experimental_features.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export type ServerlessExperimentalFeatures = Record<
+  keyof typeof allowedExperimentalValues,
+  boolean
+>;
+
+/**
+ * A list of allowed values that can be used in `xpack.dataUsage.enableExperimental`.
+ * This object is then used to validate and parse the value entered.
+ */
+export const allowedExperimentalValues = Object.freeze({
+  /**
+   * <Add a description of the feature here>
+   *
+   * [This is a fake feature key to showcase how to add a new serverless-specific experimental flag.
+   * It also prevents `allowedExperimentalValues` from being empty. It should be removed once a real feature is added.]
+   */
+  dataUsageDisabled: false,
```

Comment:
is this the default value?  if so I would interpret this to being on by default.

Reply:
Yeah, a bit confusing.. because it is reverse of enabled

Reply:
if disabled is false that means its on by default.  So it would need to be `dataUsageEnabled: false` or `dataUsageDisabled: true`.  That's if i understand correctly that this means it should be disabled by default.